autoPyTorch/constants.py

```diff
@@ -21,20 +23,23 @@
      'image_regression': IMAGE_REGRESSION}
 
 BINARY = 10
```

Comment:
Maybe we could add a comment here, saying that naming is the same as sklearns type_of_target (https://scikit-learn.org/stable/modules/generated/sklearn.utils.multiclass.type_of_target.html)